spirv_glsl.hpp

```diff
@@ -19,6 +19,7 @@
 
 #include "GLSL.std.450.h"
 #include "spirv_cross.hpp"
+#include <array>
```

Comment:
Is this needed?

Reply:
yes, used in Supporter class ::Result

Reply:
return type of the subgroup helper (ordered list of extensions to use)